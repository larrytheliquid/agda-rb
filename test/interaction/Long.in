top_command (cmd_write_highlighting_info currentFile "./highlighting.tmp")
top_command (cmd_load currentFile [])
top_command (cmd_write_highlighting_info currentFile "./highlighting.tmp")
top_command (cmd_compile MAlonzo currentFile [])
top_command cmd_constraints
top_command (cmd_load currentFile [])
top_command cmd_constraints
top_command cmd_metas
-- Note that cmd_make_case does not update the set of goals.
goal_command 0 cmd_make_case "n"
goal_command 0 cmd_refine_or_intro "Just"
goal_command 7 cmd_refine_or_intro ""
goal_command 4 cmd_auto ""
goal_command 3 (cmd_infer Instantiated) "D"
goal_command 7 (cmd_context Normalised) ""
goal_command 7 (cmd_goal_type Normalised) ""
goal_command 7 (cmd_goal_type_context Normalised) ""
goal_command 7 (cmd_goal_type_context_infer Normalised) "n"
goal_command 3 cmd_show_module_contents "D"
goal_command 3 cmd_intro ""
goal_command 6 cmd_refine "Just"
goal_command 8 (cmd_compute False) "s′ z"
goal_command 8 (cmd_compute True) "s′ z"
goal_command 8 cmd_give "d"
-- Note that cmd_solveAll does not update the set of goals.
top_command cmd_solveAll
goal_command 1 cmd_give "s z"
top_command (cmd_compute_toplevel False "eval throw")
goal_command 1 cmd_give "Nothing"
goal_command 7 cmd_give "n"
top_command (cmd_infer_toplevel Normalised "eval throw")
top_command (cmd_compute_toplevel False "eval throw")
top_command (cmd_compute_toplevel True "s′ n")
top_command (cmd_compute_toplevel True "s′ z")
top_command (cmd_show_module_contents_toplevel "D")
top_command (cmd_compile MAlonzo currentFile [])
top_command (cmd_compile Epic currentFile [])
