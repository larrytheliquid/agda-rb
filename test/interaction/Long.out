agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?0 : Maybe Nat
?1 : Maybe Nat
?2 : Nat
?3 : D ?2
?4 : D z
?5 : Set
?6 : Maybe ?5
")
agda2_mode_code (agda2-goals-action '(0 1 2 3 4 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-goals-action '(0 1 2 3 4 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Error*"
                 "
Unsolved metas at the following locations:
  Long.agda:21,16-17
  Long.agda:22,16-17
  Long.agda:27,9-13
  Long.agda:28,7-11
  Long.agda:31,7-11
  Long.agda:33,13-17
  Long.agda:34,7-11")
agda2_mode_code (agda2-status-action "")
*** Exception: ExitFailure 1
agda2_mode_code (agda2-info-action
                 "*Error*"
                 "
Error: First load the file.")
agda2_mode_code (agda2-status-action "")
*** Exception: ExitFailure 1
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?0 : Maybe Nat
?1 : Maybe Nat
?2 : Nat
?3 : D ?2
?4 : D z
?5 : Set
?6 : Maybe ?5
")
agda2_mode_code (agda2-goals-action '(0 1 2 3 4 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Constraints*" "")
agda2_mode_code (agda2-goals-action '(0 1 2 3 4 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?0 : Maybe Nat
?1 : Maybe Nat
?2 : Nat
?3 : D ?2
?4 : D z
?5 : Set
?6 : Maybe ?5
")
agda2_mode_code (agda2-goals-action '(0 1 2 3 4 5 6))
agda2_mode_code (last . (agda2-make-case-action
                         '("eval (val z) = ?" "eval (val (s n)) = ?")))
agda2_mode_code (agda2-goals-action '(0 1 2 3 4 5 6))
agda2_mode_code (agda2-give-action 0 "Just ?")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?1 : Maybe Nat
?2 : Nat
?3 : D ?2
?4 : D z
?5 : Set
?6 : Maybe ?5
?7 : Nat
")
agda2_mode_code (agda2-goals-action '(7 1 2 3 4 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Intro*"
                 "Don't know which constructor to introduce of z or s")
agda2_mode_code (agda2-goals-action '(7 1 2 3 4 5 6))
agda2_mode_code (agda2-give-action 4 "d")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?1 : Maybe Nat
?2 : Nat
?3 : D ?2
?5 : Set
?6 : Maybe ?5
?7 : Nat
")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Inferred Type*" "Nat → Set")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Context*" "n : Nat")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Current Goal*" "Nat")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Goal type etc.*"
                 "Goal: Nat
————————————————————————————————————————————————————————————
n : Nat")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Goal type etc.*"
                 "Goal: Nat
Have: Nat
————————————————————————————————————————————————————————————
n : Nat")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Module contents*"
                 "Modules
Names
  d : D z")
agda2_mode_code (agda2-goals-action '(7 1 2 3 5 6))
agda2_mode_code (agda2-give-action 3 "d")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?1 : Maybe Nat
?2 : Nat
?5 : Set
?6 : Maybe ?5
?7 : Nat
")
agda2_mode_code (agda2-goals-action '(7 1 2 5 6))
agda2_mode_code (agda2-give-action 6 "Just ?")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?1 : Maybe Nat
?2 : Nat
?5 : Set
?7 : Nat
?8 : ?5
")
agda2_mode_code (agda2-goals-action '(7 1 2 5 8))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Normal Form*" "s′ z")
agda2_mode_code (agda2-goals-action '(7 1 2 5 8))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Normal Form*" "s z")
agda2_mode_code (agda2-goals-action '(7 1 2 5 8))
agda2_mode_code (agda2-give-action 8 "d")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?1 : Maybe Nat
?2 : Nat
?5 : Set
?7 : Nat
")
agda2_mode_code (agda2-goals-action '(7 1 2 5))
agda2_mode_code (last . (agda2-solveAll-action '(2 "z" 5 "(D z)")))
agda2_mode_code (agda2-goals-action '(7 1 2 5))
agda2_mode_code (agda2-info-action
                 "*Error*"
                 "1,1-4
Nat !=< Maybe Nat of type Set
when checking that the expression s z has type Maybe Nat")
agda2_mode_code (agda2-status-action "")
*** Exception: ExitFailure 1
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Normal Form*" "?1")
agda2_mode_code (agda2-goals-action '(7 1 2 5))
agda2_mode_code (agda2-give-action 1 "Nothing")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?2 : Nat
?5 : Set
?7 : Nat
")
agda2_mode_code (agda2-goals-action '(7 2 5))
agda2_mode_code (agda2-give-action 7 "n")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?2 : Nat
?5 : Set
")
agda2_mode_code (agda2-goals-action '(2 5))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Inferred Type*" "Maybe Nat")
agda2_mode_code (agda2-goals-action '(2 5))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Normal Form*" "Nothing")
agda2_mode_code (agda2-goals-action '(2 5))
agda2_mode_code (agda2-info-action
                 "*Error*"
                 "1,4-5
Not in scope:
  n at 1,4-5
when scope checking n")
agda2_mode_code (agda2-status-action "")
*** Exception: ExitFailure 1
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*Normal Form*" "s z")
agda2_mode_code (agda2-goals-action '(2 5))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Module contents*"
                 "Modules
Names
  d : D z")
agda2_mode_code (agda2-goals-action '(2 5))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Error*"
                 "
Unsolved metas at the following locations:
  Long.agda:21,16-17
  Long.agda:22,16-17
  Long.agda:27,9-13
  Long.agda:28,7-11
  Long.agda:31,7-11
  Long.agda:33,13-17
  Long.agda:34,7-11")
agda2_mode_code (agda2-status-action "")
*** Exception: ExitFailure 1
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Error*"
                 "
Unsolved metas at the following locations:
  Long.agda:21,16-17
  Long.agda:22,16-17
  Long.agda:27,9-13
  Long.agda:28,7-11
  Long.agda:31,7-11
  Long.agda:33,13-17
  Long.agda:34,7-11")
agda2_mode_code (agda2-status-action "")
*** Exception: ExitFailure 1
